src/android/io/sentry/SentryCordova.java

```diff
@@ -1,47 +1,370 @@
+          options.setDsn(dsn);
+        } else {
+          // SentryAndroid needs an empty string fallback for the dsn.
+          options.setDsn("");
+        }
+        if (jsonOptions.has("debug") && jsonOptions.getBoolean("debug")) {
+          options.setDebug(true);
+        }
+        if (jsonOptions.has("maxBreadcrumbs")) {
+          options.setMaxBreadcrumbs(jsonOptions.getInt("maxBreadcrumbs"));
+        }
+        if (jsonOptions.has("environment") && jsonOptions.getString("environment") != null) {
+          options.setEnvironment(jsonOptions.getString("environment"));
+        }
+        if (jsonOptions.has("release") && jsonOptions.getString("release") != null) {
+          options.setRelease(jsonOptions.getString("release"));
+        }
+        if (jsonOptions.has("dist") && jsonOptions.getString("dist") != null) {
+          options.setDist(jsonOptions.getString("dist"));
+        }
+        if (jsonOptions.has("enableAutoSessionTracking")) {
+          options.setEnableSessionTracking(jsonOptions.getBoolean("enableAutoSessionTracking"));
+        }
+        if (jsonOptions.has("sessionTrackingIntervalMillis")) {
+          options.setSessionTrackingIntervalMillis(jsonOptions.getInt("sessionTrackingIntervalMillis"));
+        }
+        if (jsonOptions.has("enableNdkScopeSync")) {
+          options.setEnableScopeSync(jsonOptions.getBoolean("enableNdkScopeSync"));
+        }
+        if (jsonOptions.has("attachStacktrace")) {
+          options.setAttachStacktrace(jsonOptions.getBoolean("attachStacktrace"));
+        }
+        if (jsonOptions.has("attachThreads")) {
+          // JS use top level stacktraces and android attaches Threads which
+          // hides them so by default we hide.
+          options.setAttachThreads(jsonOptions.getBoolean("attachThreads"));
+        }
 
-          SentryAndroid.init(this.cordova.getActivity().getApplicationContext(), options -> {
-            options.setDsn(dsn);
-          });
+        if (jsonOptions.has("enableNativeCrashHandling") && !jsonOptions.getBoolean("enableNativeCrashHandling")) {
+          final List<Integration> integrations = options.getIntegrations();
+          for (final Integration integration : integrations) {
+            if (integration instanceof UncaughtExceptionHandlerIntegration || integration instanceof AnrIntegration || integration instanceof NdkIntegration) {
+              integrations.remove(integration);
+            }
+          }
+        }
 
-          callbackContext.sendPluginResult(new PluginResult(Status.OK, true));
-          break;
-        default:
-          callbackContext.sendPluginResult(new PluginResult(Status.ERROR, "not implemented"));
-          break;
+        logger.info(String.format("Native Integrations '%s'", options.getIntegrations().toString()));
+        sentryOptions = options;
+      } catch (JSONException e) {
+        logger.info("Error parsing options JSON sent over native bridge.");
```

Comment:
```suggestion
        logger.error("Error parsing options JSON sent over native bridge.");
```

Reply:
Will make this `.severe` https://developer.android.com/reference/java/util/logging/Logger#severe(java.lang.String)